src/index.html

```diff
@@ -11,7 +11,175 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="get"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <label class="formField">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="formField">
+          Name:
+          <input
+            type="text"
+            name="name"
+          >
+        </label>
+
+        <label class="formField">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+
+        <label class="formField">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+          >
+        </label>
+
+        <label class="formField">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="acceptTerms"
+          >
+        </label>
+      </fieldset>
+
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <label class="formField">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+
+        <label class="formField">
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+            minlength="8"
+            maxlength="20"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+
+        <div class="formField">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="loveCats"
+            >
+              Yes
+          </label>
+          <label>
+            <input
+              type="radio"
+              name="loveCats"
+            >
+            No
+          </label>
+        </div>
+
+        <label class="formField">
+          What is your favorite color?
+          <input
+          type="color"
+          name="color"
+          >
```

Comment:
pay attention to attribute formatting, next time I will reject task if I see it
```suggestion
          <input
            type="color"
            name="color"
          >
```